src/Native/System.Native/pal_networking.h

```diff
@@ -40,43 +44,65 @@ enum GetHostErrorCodes
     PAL_NO_ADDRESS = PAL_NO_DATA,
 };
 
-struct PalIpAddress
+/**
+ * Address families recognized by {Get,Set}AddressFamily
```

Comment:
Yes--I meant to have a comment (and perhaps assertions in the managed code for validation).